library/Mockery/Expectation.php

```diff
@@ -331,7 +340,7 @@ protected function _matchArg($expected, &$actual)
         if ($expected instanceof \Mockery\Matcher\MatcherAbstract) {
             return $expected->match($actual);
         }
-        if (is_a($expected, '\Hamcrest\Matcher') || is_a($expected, '\Hamcrest_Matcher')) {
+        if ($expected instanceof \Hamcrest\Matcher || $expected instanceof \Hamcrest_Matcher) {
```

Comment:
The `is_a` was used, because we're unsure if mentioned class is present. Probably now we can use `instanceof` because Hamcrest is a required dependency, but this isn't the topic of this PR.